scripts/integration_test_local.sh

```diff
@@ -62,14 +62,14 @@ function create_real_docker_config_json() {
 
 function back_up_kube_config() {
   # don't let minikube overwrite user's real k8s kubeconfig
-  local backed_up_kube_config=''
-  if ls "$KUBE_CONFIG_PATH"
+  retval=1
+  if ls "$KUBE_CONFIG_PATH" &>/dev/null
   then
-    echo "Backing up $KUBE_CONFIG_PATH ..."
     mv "$KUBE_CONFIG_PATH" "$KUBE_CONFIG_BACKUP_PATH"
-    backed_up_kube_config='true'
+    echo "Backing up $KUBE_CONFIG_PATH..."
+    retval=0
   fi
-  echo "$backed_up_kube_config"
+  return "$retval"
```

Comment:
been relying on `echo` to return backed_up_kube_config flag, problem is all `echo`s are returned. so switching to `return`.